app/src/main/res/layout/frag_filesystem_edit.xml

```diff
@@ -42,4 +42,12 @@
         app:layout_constraintStart_toEndOf="@id/text_filesystem_type"
         app:layout_constraintTop_toBottomOf="@id/text_input_layout" />
 
+    <Button
```

Comment:
I think this action should be moved into the filesystem list fragment and be displayed in the toolbar. Clicking it could navigate to a new fragment that handles filesystem restoration.

Reply:
Ok. As mentioned above I would like to do this as fragment with list of tracked backups.

Reply:
Yeah, that's a good idea.